tests/common.test.js

```diff
@@ -0,0 +1,272 @@
+import { expect, describe, test, vi, beforeEach } from 'vitest';
+import { oneOffMeeting, notGoogleMeetMeeting } from './data';
+import { 
+    convertedMeeting, 
+    isMeetingInFuture, 
+    isGoogleMeet,
+    shouldReplace,
+    getDisplayTimeFor,
+    getCalendars,
+    getMeetingsFor,
+    getEndOfDay
+} from '../src/common';
+
+test('convert one off meeting', () => {
+    let expected = {
+        id: oneOffMeeting.id,
+        meet: oneOffMeeting.hangoutLink,
+        dateTime: oneOffMeeting.start.dateTime,
+        name: oneOffMeeting.summary,
+        allDay: false,
+        updatedDateTime: oneOffMeeting.updated,
+    }
+
+    expect(convertedMeeting(oneOffMeeting)).toStrictEqual(expected);
+});
+
+describe('meeting is in future', () => {
+    describe('for one off meeting with same date', () => {
+        const meeting = {
+            end: {
+                dateTime: "2023-05-16T14:30:00+01:00"
+            },
+            start: {
+                dateTime: "2023-05-16T13:00:00+01:00"
+            }
+        };
+    
+        test('should be true by hour', () => {
```

Comment:
does this test name make sense? I know what they're doing but I'd rename these tests